code/sum_of_digits/sum_of_digits.cpp

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
```

Comment:
This is calculating sum of 2 numbers not sum of digits. Kindly correct your code to get this merged 👍